bedita-app/webroot/js/libs/richtexteditors/ckeditor/styles.js

```diff
@@ -52,6 +52,12 @@ CKEDITOR.stylesSet.add( 'default', [
 		attributes: { 'class': 'raccordo' }
 	},
 
+	{		
+		name: 'Nota',
+		element: 'DIV',
+		attributes: { 'id': 'note' }
+	},
+
 	/* Inline Styles */
 
 	{ name: 'Definition',		element: 'dfn'},
```

Comment:
Could you equalize the indentation? Either indenting more the lines i selected, or indenting less the lines before